config/canonical.json

```
@@ -30100,6 +30100,8 @@
     "count": 619,
     "tags": {
       "brand": "IGA",
+      "brand:wikidata": "Q5970945",
+      "brand:wikipedia": "en:IGA_(supermarkets)",
```

Remove the `_`, then it's good to merge.. 

```suggestion
      "brand:wikipedia": "en:IGA (supermarkets)",
```